Source/Basic-Conditions-And-Behaviors/Editor/UI/MenuItems/CollectedConditionMenuItem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Runtime.Serialization;
+using VRBuilder.Core.Attributes;
+using VRBuilder.Core.Conditions;
+using VRBuilder.Editor.UI.StepInspector.Menu;
+
+namespace VRBuilder.Editor.UI.Behaviors
+{
+    [DataContract(IsReference = true)]
+    [HelpLink("https://www.mindport.co/vr-builder-tutorials/creating-custom-conditions")]
+    public class CollectedConditionMenuItem : MenuItem<ICondition>
+    {
+        /// <inheritdoc />
+        public override string DisplayedName { get => "Environment/Unique Object in Collider"; }
```

Comment:
```suggestion
        public override string DisplayedName { get => "Environment/New Object in Collider"; }
```
I would personally go with "New object" rather than "Unique object" as it better conveys what it does.